src/blocks/stars.css

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+}
+
+.stars__sizing {
+  width: 16px;
+  height: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stars__sizing:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star {
+  width: 13px;
+  height: 12px;
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
```

Comment:
You're using `background-size: contain;` which is good for ensuring the star image fits within the element, but make sure that the aspect ratio of the star is consistent with the design requirements.